atomicfu/src/jvmMain/kotlin/kotlinx/atomicfu/AtomicFU.kt

```diff
@@ -72,6 +72,7 @@ public actual class AtomicRef<T> internal constructor(value: T) {
         set(value) {
             interceptor.beforeUpdate(this)
             field = value
+            trace { "value.set($value)" }
```

Comment:
This line adds a lot of string allocations to any atomic-using code even when trace is not used (which is default). I'd suggest to change this line to:
```            
if (trace !== TraceBase.None) trace { "value.set($value)" }
```
and do the same in all other AtomicXxx methods.